docs: apply Snyk writing-style skill to Snyk API & Web pages (#1357)
snyk/user-docs: scan-fix-and-prevent/scan-with-snyk/snyk-api-web/configure-targets/configure-api-targets/configure-postman-collection-targets.md

    var jsonData = pm.response.json();  
    pm.collectionVariables.set('user_id', jsonData.results[0].id);
    ```
2. Navigate to the request that gets the user details and add the `user_id` variable as a parameter.

> The request to get the user details looks like the following example:

Reviewer: ```suggestion
The following image shows an example of a request to get user details.
```
